makefile

```diff
@@ -41,7 +41,7 @@ lint:
 .PHONY: test
 test:
 	@echo -e "$(ATTN_COLOR)==> $@ $(NO_COLOR)"
-	@${EXT_BIN_DIR}/gotestsum --format short-verbose -- -count=1 -parallel=1 -v -coverprofile=cover.out -coverpkg=./... ./...;
+	@${EXT_BIN_DIR}/gotestsum --format short-verbose -- -coverprofile=cover.out -coverpkg=./... ./...;
```

Comment:
-count=1 is being used to not cache test results, forcing re-execution of the package.

From the go docs
```
When 'go test' runs in package list mode, 'go test' caches successful package test results to avoid unnecessary repeated running of tests. To disable test caching, use any test flag or argument other than the cacheable flags. The idiomatic way to disable test caching explicitly is to use -count=1.
```